### PR TITLE
Output Kubernetes Cluster Name

### LINKED ISSUE #233: Output Kubernetes Name
Output Kubernetes name. This can help consumer to use that information to do any dependent tasks. Such as output a pre-built command to get AKS credentials. etc.

### PR DESCRIPTION
**Pull request by vermacodes**
Fixes #233 

Changes proposed in the pull request:

Output name of the Kubernetes cluster

### CODE REVIEW

**Comment by vermacodes** [comment]
Hello @lonegunmanb Any thoughts on this PR?

File: outputs.tf [hunk 1]
```diff
@@ -49,6 +49,11 @@ output "aks_id" {
   value       = azurerm_kubernetes_cluster.main.id
 }
 
+output "aks_name" {
+  description = "The `aurerm_kubernetes-cluster`'s name"
```
Comment (lonegunmanb) [inline outputs.tf:53]: Can we add a dot in the description and readme file? Thanks!

```hcl
description = "The `aurerm_kubernetes-cluster`'s name."
```

**Review by lonegunmanb — CHANGES_REQUESTED** [review]
Almost LGTM but a tiny change request, thanks for your pr!

File: outputs.tf (hunk 1 above)
Comment (vermacodes) [reply outputs.tf]: Added.

**Review by lonegunmanb — APPROVED** [review]
LGTM, thanks for your contribution @vermacodes !

**Pull request merged**